modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFormProvider.java

```diff
@@ -41,6 +41,18 @@ private InfoFieldSet _getBasicInformationInfoFieldSet() {
 		return InfoFieldSet.builder(
 		).infoFieldSetEntry(
 			CalendarBookingInfoItemFields.titleInfoField
+		).infoFieldSetEntry(
```

Comment:
Question: 

No alphabetical order is ok?

Reply:
Yes; form providers list fields in a "user friendly" order (this order will be used when mapping fields in the page editor).